zkevm-circuits/src/table/mpt_table.rs

```diff
@@ -1,68 +1,134 @@
+use serde::{Deserialize, Serialize};
+
 use super::*;
+use crate::{
+    circuit,
+    circuit_tools::{
+        cached_region::{CachedRegion, ChallengeSet},
+        cell_manager::CellType,
+        constraint_builder::ConstraintBuilder,
+    },
+};
 
 /// The types of proofs in the MPT table
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Serialize, Deserialize)]
 pub enum MPTProofType {
+    /// Disabled
+    Disabled,
     /// Nonce updated
-    NonceMod = AccountFieldTag::Nonce as isize,
+    NonceChanged = AccountFieldTag::Nonce as isize,
     /// Balance updated
-    BalanceMod = AccountFieldTag::Balance as isize,
+    BalanceChanged = AccountFieldTag::Balance as isize,
     /// Code hash exists
-    CodeHashMod = AccountFieldTag::CodeHash as isize,
+    CodeHashExists = AccountFieldTag::CodeHash as isize,
```

Comment:
@Brechtpd At some point I thought we will need only `CodeHashExists`, but after some discussions we realised we also need `CodeHashChanged`. Is it ok to change the name?

Reply:
Oh yes of course, will make it part of the next batch of changes together with the name changes above!